src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -282,7 +282,7 @@ protected function trans($key, $domain, array $parameters = [])
     protected function flashErrors(array $errorMessages)
     {
         foreach ($errorMessages as $error) {
-            $message = is_array($error) ? $this->trans($error['key'], $error['domain'], $error['parameters']) : $error;
+            $message = is_array($error) ? $this->trans($error['key'], $error['domain'], !empty($error['parameters']) ? $error['parameters'] : []) : $error;
```

Comment:
I'm not in favor of this change, an error message should have the 3 properties, and parameters is expected to return an array.

You're hiding the fact that somewhere in the code, we have a malformed error message :/

Reply:
but is it really necessary to force the developer to add an empty parameters array when it is useless?

Reply:
I know it's unfair, in the future we will have an ErrorMessage value object with 3 public properties, but for now and because we're using PHP5 we need to be really defensive on types we manipulate.

For instance, I'm worried about your check:

```
php > var_dump(empty(true));
bool(false)
php > var_dump(empty(false));
bool(true)
php > var_dump(empty(''));
bool(true)
php > var_dump(empty([]));
bool(true)
php > var_dump(empty(['']));
bool(false)
php > var_dump(empty(new stdClass()));
bool(false)
```
Because it can hide some errors.